pkg/kanctl/actionset.go

```diff
@@ -105,6 +107,7 @@ func newActionSetCmd() *cobra.Command {
 	cmd.Flags().String(selectorNamespaceFlag, "", "namespace to apply selector on. Used along with the selector specified using --selector/-l")
 	cmd.Flags().StringSliceP(namespaceTargetsFlagName, "T", []string{}, "namespaces for the action set, comma separated list of namespaces (eg: --namespacetargets namespace1,namespace2)")
 	cmd.Flags().StringSliceP(objectsFlagName, "O", []string{}, "objects for the action set, comma separated list of object references (eg: --objects group/version/resource/namespace1/name1,group/version/resource/namespace2/name2)")
+	cmd.Flags().BoolP(waitForReadyFlagName, "w", false, "wait for resources to get in ready state")
```

Comment:
The `wait` flag seems a bit ambiguous. Is it for waiting for actionset to complete, or repo server to be ready or resources to be ready? We can have something `--wait-for-reposerver`?

Reply:
Done !